packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -689,3 +693,22 @@ getCollectionHooks("Posts").editSync.add(async function removeFrontpageDate(
   }
   return modifier;
 });
+
+getCollectionHooks("Posts").updateAfter.add(async function createJargonTermsCallback (_post, {newDocument, oldDocument, context}: UpdateCallbackProperties<"Posts">) {
+  const currentUser = context.currentUser
+  if (!currentUser) return _post
+  if (currentUser._id !== newDocument.userId) return _post
+  if (!userCanCreateAndEditJargonTerms(currentUser)) return _post
+
+  const newContents = await Revisions.findOne({_id: newDocument.contents_latest})
+  if (!newContents?.html) {
+    return _post;
+  }
+  const existingJargon = await JargonTerms.find({postId: newDocument._id}).fetch()
```

Comment:
You can pull the collection `JargonTerms` from the context and don't need to import it